qa-pipelines/tasks/cf-teardown.sh

```diff
@@ -3,16 +3,22 @@ set -o errexit -o nounset
 
 # Set kube config from pool
 source "ci/qa-pipelines/tasks/lib/prepare-kubeconfig.sh"
+source "ci/qa-pipelines/tasks/lib/klog-collection.sh"
+
 set -o allexport
 CF_NAMESPACE=scf
 UAA_NAMESPACE=uaa
 set +o allexport
 
-for namespace in "$CF_NAMESPACE" "$UAA_NAMESPACE" ; do
+set -- "${CF_NAMESPACE}" "${UAA_NAMESPACE}"
```

Comment:
Should probably use your own array variable rather than clobbering `$@`; this makes it hard later if we want to pass in positional arguments.